packages/backend/src/paypalapi/paypalClient.ts

```diff
@@ -8,7 +8,7 @@ export const paypalClient = createClient<billingPaths & catalogsPaths>({ baseUrl
 async function getAccessToken() {
   const clientId = process.env.NEXT_PUBLIC_PAYPAL_CLIENT_ID
   const clientSecret = process.env.PAYPAL_CLIENT_SECRET
-  const oauthTokenUrl = process.env.PAYPAL_URL + 'v1/oauth2/token'
+  const oauthTokenUrl = process.env.PAYPAL_URL + '/v1/oauth2/token'
```

Comment:
Could we make it more safe to be sure that under no circumstances the middle '//' will never happen?